setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='xbundle',
+    version='0.1.0',
+    packages=['xbundle'],
+    scripts=['bin/xbundle_convert.py'],
```

Comment:
Do you know if we need to keep the name, or can we switch to just having 'xbundle_convert' and using an entrypoint like:
```python
    entry_points={'console_scripts': [
        'xbundle_convert = xbundle_convert:__main__',
    ]},
```
also, important note, I have no idea if that entrypoint would actually work by calling __main__, but would be nice to use that instead of scripts